src/test/ui/type-alias-impl-trait/issue-65679.rs

```diff
@@ -0,0 +1,11 @@
+// edition:2018
+// run-pass
+
```

Comment:
Could you also provide a description of the problem + rename the file to *also* include a brief description of the problem, e.g. `issue-65679-inst-opaque-ty-from-val-twice.rs`?